html-web-page/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <h1>Jelly of the Month Club</h1>
+  <h2>Make all your friends jelly of your stock of jelly!</h2>
+  <p>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
+    aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis
+    aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint
+    occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+  </p>
+  <table>
+    <tr>
+      <th></th>
+      <th>Basic Plan</th>
+      <th>Deluxe Plan</th>
+      <th>Jelly Giant Plan</th>
+    </tr>
+    <tr>
+      <td>Three new curated jellies per month</td>
+      <td>&check;</td>
+      <td>&check;</td>
+      <td>&check;</td>
+    </tr>
+    <tr>
+      <td>Monthly jellies come with commemorative mason jars</td>
+      <td></td>
+      <td>&check;</td>
+      <td>&check;</td>
+    </tr>
+    <tr>
+      <td>Free bonus jar of preserves bi-monthly</td>
+      <td></td>
+      <td>&check;</td>
+      <td>&check;</td>
+    </tr>
+    <tr>
+      <td>Free yearly tickets to Jelly Con</td>
+      <td></td>
+      <td></td>
+      <td>&check;</td>
+    </tr>
+    <tr>
+      <td>Monthly Pricing</td>
+      <td>$30</td>
+      <td>$50</td>
+      <td>$60</td>
+    </tr>
+  </table>
+  <p>Like what you see? Become a part of the JOTMC family today!</p>
+  <form>
+    <label for="name">
+      <input id="name" placeholder="Your Name">
+    </label>
+    <label for="plan">
+      <select>
+        <option>Select Your Plan:</option>
+      </select>
+    </label>
+    <label for="submit">
+      <input id="submit" type="submit" value="Submit" />
+    </label>
```

Comment:
Since there is no text labelling these form control elements, no `label` elements are necessary here.